aws_adfs/_version.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.3'
+__version__ = '0.3.5'
```

Comment:
I've changed the versioning mechanism from file base to git based - you can omit changes in this file.